pkg/api/team.go

```diff
@@ -180,6 +180,42 @@ func (c *client) setupTeamRoutes() {
 		}
 		ctx.Redirect(http.StatusSeeOther, "/oversikt")
 	})
+
+	c.router.GET("/team/:slug/events", func(ctx *gin.Context) {
+		teamSlug := ctx.Param("slug")
+		team, err := c.repo.TeamBySlugGet(ctx, teamSlug)
+		if err != nil {
+			if errors.Is(err, sql.ErrNoRows) {
```

Comment:
Kom på noe, vi har jo en sjekk for om du er medlem av teamet du prøver å nå. Den vil jo hindre at dette noen gang vil oppstå. Betyr jo at vi egentlig ikke trenger denne type sjekker 🤔